thing-url-adapter.js

```diff
@@ -264,8 +263,27 @@ class ThingURLDevice extends Device {
     if (this.closing) {
       return;
     }
+    // if we have an entry for the wss url stub in jwt_auth
+    // add auth to query string
+    let auth = '';
+
+    for (const i in this.adapter.authData) {
```

Comment:
This would be a little cleaner:

`for (const [url, authData] of Object.entries(this.adapter.authData))`

Reply:
removed